recipes/zlib-ng/all/test_package/test_package.c

```diff
@@ -0,0 +1,15 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#ifdef ZLIB_COMPAT
+#  include "zlib.h"
+#  define ZLIB_VERSION zlibVersion
+#else
+#  include "zlib-ng.h"
+#  define ZLIB_VERSION zlibng_version
+#endif
+
+int main(void) {
+    printf("ZLIB NG VERSION: %s\n", ZLIB_VERSION());
+    return EXIT_SUCCESS;
+}
```

Comment:
```suggestion
}

```
Conan hooks requires empty end-line